.dockerignore

```diff
@@ -10,3 +10,4 @@ packages/generator-joplin
 packages/plugin-repo-cli
 packages/server/db-*.sqlite
 packages/server/temp
+.yarn/cache
```

Comment:
I think this should stay, it helps with any local re-runs and really has no harm .. the yarn cache should not be added to the docker context anyways ;)

Reply:
Hmm, I don't know? If we put the cache on the image, then Yarn won't need to download the package which perhaps would make the process faster?

Reply:
but the caches should not be pulled from your local machine, they could contain files for the wrong architectures or whatnot so its not ideal .. its a best practice to not include this kind of stuff in the docker image (this also breaks actual layer caching pretty bad .. since we copy in the .yarn folder pretty early on .. any local changes to that cache (ie any time you run yarn install locally) .. will cause the docker layer cache to get reset .. which seems to be worse for timing :)

Reply:
even if yarn handles architecture caching properly .. its still causing the cache to get reset each time a new cached file is added in your local which is not ideal .. and you don't want to actually SHIP the cache in the image .. it makes the image unnecessarily large and are just junk files for the actual running of the server

Reply:
The general rule of thumb is that only "source" files should be pulled into the docker context (I'd also have added all the *.js *.js.map etc files that are generated by the build process .. these are expected to be re-generated when you build the packages anyways .. but that's a change for another day ;)

Reply:
Right, it's true it shouldn't be included. Yarn has this "zero-install" technique where all cache is committed to the repo, however we aren't using this so it probably also wouldn't make sense to add them to the docker context.